### DISCUSSION TITLE
How to create a client having a static password ?

### DISCUSSION THREAD

**Discussion by nicolasduminil** [keycloak/keycloak#25879] [Archive]
Hello,
Using `quay.io/keycloak/keycloak:latest`, I need to create a client having a static, not generated, password, for example `mypassword1`, using the `kcadm.sh` utility. I don't seem to manage to find how to do that. 
Many thanks in advance for your help.
Kind regards,
Nicolas

**Comment by souravs17031999** [comment]
Check if this can help 
https://gist.github.com/thomasdarimont/bb702bd1160eb200147cf1bee1c1f7ed

**Reply by nicolasduminil** [reply]
Yeah, it helped, thanks.
